docs/content/stable/explore/query-1-performance/explain-analyze.md

```diff
@@ -41,8 +41,8 @@ The output is effectively the plan tree followed by a summary of timing and coun
 - **Timings**: At the end of the plan tree, YugabyteDB will add multiple time taken metrics when the [DIST](../../../api/ysql/the-sql-language/statements/perf_explain/#dist) option is specified. These are aggregate times across all plan nodes. Some of them are,
     - _Planning Time_: The time taken in milliseconds for the [query planner](../../../architecture/query-layer/#planner) to design the plan tree. Usually this value is higher the first time a query is executed. During subsequent runs, this time will be low as the plans are cached and re-used.
     - _Execution Time_: The total time taken in milliseconds for the query to execute. This includes all table/index read/write times.
-    - _Storage Read Execution Time_: The sum of all round-trip times taken to read tables and indexes.
-    - _Storage Write Execution Time_: The sum of all round-trip times taken to flush the write requests to tables and indexes.
+    - _Storage Read Execution Time_: The total time spent by the YSQL query layer in waiting to receive data from the DocDB storage layer. This is measured as the sum of wait times for individual RPCs to read tables and indexes. The YSQL query layer often pipelines requests to read data from the storage layer, and so the _Storage Read Execution Time_ is often lower than the sum of round-trips times to read tables and indexes.
```

Comment:
```suggestion
    - _Storage Read Execution Time_: The total time spent by the YSQL query layer waiting to receive data from the DocDB storage layer. This is measured as the sum of wait times for individual RPCs to read tables and indexes. Because the YSQL query layer often pipelines requests to read data from the storage layer, the _Storage Read Execution Time_ is often lower than the sum of the times for round-trips to read tables and indexes.
```